addons/hide-stage/style.css

```diff
@@ -52,3 +52,7 @@
   /* makes small and large stage buttons appear unselected */
   filter: var(--editorDarkMode-accent-desaturateFilter, saturate(0));
 }
+
+.sa-stage-hidden .scratchEyedropper {
+  display: none;
+}
```

Comment:
We don't want to repeat our mistakes here. Scratch doesn't assume that the eyedropper is showing, does it?

Reply:
This bug happens because Scratch uses the stage's width and height for calculations. I don't think Scratch ever calls `clientWidth` on the eye dropper element.

Reply:
Shouldn't be an issue